redirects.txt

```diff
@@ -172,3 +172,6 @@ discuss/email_servers.rst discuss/advanced/email_servers.rst               # (#6
 discuss/plan_activities.rst discuss/overview/plan_activities.rst           # (#655)
 discuss/team_communication.rst discuss/overview/team_communication.rst     # (#655)
 discuss/overview.rst discuss/overview/get_started.rst                      # (#655)
+
+crm/optimize/onsip.rst general/voip/onsip.rst                                                       # crm/optimize/* --> general/voip/*
+crm/optimize/setup.rst general/voip/asterisk.rst                                                    # crm/optimize/setup --> general/voip/asterisk
```

Comment:
```suggestion
crm/optimize/onsip.rst general/voip/onsip.rst      # crm/optimize/* --> general/voip/*
crm/optimize/setup.rst general/voip/asterisk.rst  # crm/optimize/setup --> general/voip/asterisk
```

Reply:
why so, @AntoineVDV ? We always left some spaces in that doc. Here, I made the comments appear after the 100th character.

Reply:
Ok but why? :stuck_out_tongue: 
If we left some space before it was for the sake of clarity. Here you add an entire page width of space to conform with a guideline that doesn't exist.

Reply:
@AntoineVDV 
I asked the question but had already done the changes locally :)
I believe the PR is ready to be merged.